Nav fix

kart-c opened the pull request:
## Nav Fix

- Navigation in desktop sticks to the left throughout scroll
- Added navigation to hamburger menu in mobile view
- Clicking on logo will guide the user to landing page

 CSB for reference https://zivxx.csb.app/

kart-c committed:
fix: side navigation now visible in mobile view

kart-c committed:
feat: clicking on logo in navbar guides user to landing page

kart-c committed:
feat: all navigation links visible in mobile view

kart-c committed:
feat: aside sticks to the left throughout scroll

Husain01 reviewed (approved):
Nice work with the sidebar

Kevin-Solomon commented:
Looks good , Kartik .
One small suggestion if you could add a background of grey on the main page when the mobile-nav is active.

kart-c commented:
> Looks good , Kartik . One small suggestion if you could add a background of grey on the main page when the mobile-nav is active.

Thank you for noticing, I will add it right away

kart-c committed:
fix: backdrop was not visible on landing page

js was searching trying to add classes on aside menu, which does not exist on landing page
